index.html

```diff
@@ -36,7 +36,7 @@
 			function populateZoomControl(selectID, sourceID, fieldName, layerName) {
 				polygons = getPolygons(sourceID, fieldName);
 				var select = document.getElementById(selectID);
-				select.options[0] = new Option(layerName);
+				select.options[0] = new Option(layerName, "-108,25,-88,37");
```

Comment:
This is the only functional change in this PR.  Where all the other options in that dropdown get a bounding box set dynamically from the individual district's polygon, the whole-state option is hardcoded to a bbox that is slightly bigger than the one you set for the map.  This just makes it zoom out as far as it's allowed to.